src/hooks.server.ts

```diff
@@ -1,6 +1,21 @@
 import type { Handle } from "@sveltejs/kit";
+import { KV_URL } from "$env/static/private";
+import { sequence } from "@sveltejs/kit/hooks";
+import { createClient } from "$lib/services/server/kv";
 
-export const handle = (async ({ event, resolve }) => {
+const handleSetupRedisClient = (async ({ event, resolve }) => {
```

Comment:
When running on Vercel (as opposed to local development) there's no real distinction between "when the app starts" and "when it handles a request" because we're using serverless functions, so the app starts and runs to completion for each request. We can't persist a client between different invocations of a serverless function.